### PR TITLE
Update EnumWrapper

### PR DESCRIPTION
**Pull request by Morlic-fo**
- Add Happiness Meter Enums
- Reorder entries according to Enums.h
- Remove outdated starlaneSpeed entries.

### CODE REVIEW

File: python/EnumWrapper.cpp [hunk 1]
```diff
@@ -149,12 +150,11 @@ namespace FreeOrionPython {
             .value("stealth",           PC_STEALTH)
             .value("fuel",              PC_FUEL)
             .value("colony",            PC_COLONY)
-            .value("starlaneSpeed",     PC_SPEED) // TODO: Remove this after some transition period
```
Comment (geoffthemedio) [inline python/EnumWrapper.cpp:152]: you're sure nothing references it in Python code?

Comment (Morlic-fo) [inline python/EnumWrapper.cpp:152]: At least I can't find any uses when searching all files.

**Comment by Cjkjvfnby** [comment]
If we store any integer in savegame and compare it with enum it will break savegame. I hope we did not do that.

Maybe sore enums by abc?

I don't think that `freeOrionAIInterface.py` should be part of PR. Last time I just pushed it directly to master. It is generated by script and not used directly in game code.

**Comment by Vezzra** [comment]
Builds and runs ok on OSX.

**Comment by Cjkjvfnby** [comment]
@Morlic-fo, please rebase, I have updated `freeOrionAIInterface.py`

**Comment by Morlic-fo** [comment]
I excluded the update to `freeOrionAIInterface.py`.

**Comment by geoffthemedio** [comment]
lacking further comment, assuming this is fine...

**Pull request merged**